src/app/components/feed/FeedItemMediaList.js

```diff
@@ -1,18 +1,19 @@
-/* eslint-disable react/sort-prop-types */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { QueryRenderer, graphql } from 'react-relay/compat';
 import Relay from 'react-relay/classic';
-import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { injectIntl } from 'react-intl';
```

Comment:
is there a reason to keep `injectIntl` if we're not using the `intl` prop?

Reply:
same as previous comment, resolved now